cmd/main.go

```diff
@@ -50,15 +49,14 @@ var (
 	port         = flag.Int("port", 12345, "Port on which to run the server.")
 	useALTS      = flag.Bool("use_alts", false, "Whether to use ALTS server authentication")
 	mixerProject = flag.String("mixer_project", "", "The GCP project to run the mixer instance.")
-	// Bigtable and BigQuery project
-	storeProject = flag.String("store_project", "", "GCP project stores Bigtable and BigQuery.")
 	// BigQuery (Sparql)
 	useBigquery = flag.Bool("use_bigquery", true, "Use Bigquery to serve Sparql Query")
 	bqDataset   = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
 	schemaPath  = flag.String("schema_path", "", "The directory that contains the schema mapping files")
 	// Base Bigtable Cache
-	useBaseBt         = flag.Bool("use_base_bt", true, "Use base bigtable cache")
-	importGroupTables = flag.String("import_group_tables", "", "Newline separated list of import group tables")
+	useBaseBt      = flag.Bool("use_base_bt", true, "Use base bigtable cache")
+	baseBigtable   = flag.String("base_bigtable", "", "Yaml file containing information for base Bigtable")
+	customBigtable = flag.String("custom_bigtable", "", "Yaml file containing information for custom Bigtable")
```

Comment:
nit:  wonder if you mean bigtable "instance" here?  if so, baseBigtableInstance, customBigtableInstance

Reply:
I now realize it is a group of bigtables in a specific instance in a cloud project.  should we use a common terminology for it, rather than just "bigtable" (which can be confused with a specific BT)?  bigtable_group?

Reply:
Updated to "base_bigtable_info" and "custom_bigtable_info". As this could hold other information like pubsub info for branch cache.